src/main/flight/pid.c

```diff
@@ -52,6 +52,10 @@
 
 extern uint8_t motorCount;
 extern float dT;
+extern bool motorLimitReached;
+bool antiWindupProtection;
+
+#define PREVENT_WINDUP(x,y,z) { if (z) {if (ABS(x) > ABS(y)) { if (x < 0) { x = -ABS(y); }  else { x = ABS(y); } } } else { y = x; }}
```

Comment:
To much is hidden in this macro - it conditionally reverses direction of input/output arguments ...